src/attack.c

```diff
@@ -627,6 +627,47 @@ vhosts_to_str (GSList *list)
   return g_string_free (string, FALSE);
 }
 
+/**
+ * @brief Check if any deprecated prefs are in pref table and print warning.
+ *
+ * @return True if deprecated prefs are in preference table. Else False.
+ */
+gboolean
```

Comment:
Shouldn't this be static? Shouldn't be just a void function? the return value is not used, and the message is sent directly from the function.